From dc60d67002d5d829219ef0b5a53fa45cf86cd282 Mon Sep 17 00:00:00 2001
From: Hyun Kwon <hyun.kwon@xilinx.com>
Date: Mon, 11 Jun 2018 17:50:20 +0530
Subject: [PATCH 0365/1851] drm: xlnx: vtc: Block comments using a trailing */
 on a separate line

commit b883e66ac75d1baaf44e576802367cc7a8f5056a from
https://github.com/Xilinx/linux-xlnx.git

Fix the comments to avoid following warnings from checkpatch:

./scripts/checkpatch.pl --max-line-length 120 -f xilinx_vtc.c
WARNING: Block comments use * on subsequent lines
+#define VTC_XVXHOX_HEND_SHIFT	16		/* horizontal offset end
+						   shift */

WARNING: Block comments use a trailing */ on a separate line
+						   shift */

WARNING: Block comments use * on subsequent lines
+#define VTC_IXR_HSL	(1 << 19)	/* horizontal sync signal
+						   lock */

WARNING: Block comments use a trailing */ on a separate line
+						   lock */

WARNING: Block comments use * on subsequent lines
+#define VTC_IXR_HBL	(1 << 17)	/* horizontal blank signal
+						   lock */

WARNING: Block comments use a trailing */ on a separate line
+						   lock */

total: 0 errors, 6 warnings, 589 lines checked

Signed-off-by: Hyun Kwon <hyun.kwon@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/gpu/drm/xlnx/xlnx_vtc.c | 13 +++++--------
 1 file changed, 5 insertions(+), 8 deletions(-)

diff --git a/drivers/gpu/drm/xlnx/xlnx_vtc.c b/drivers/gpu/drm/xlnx/xlnx_vtc.c
index 019c0f7a5455..cfd778b27aa0 100644
--- a/drivers/gpu/drm/xlnx/xlnx_vtc.c
+++ b/drivers/gpu/drm/xlnx/xlnx_vtc.c
@@ -91,10 +91,9 @@
 #define VTC_GV1_SYNCSTART_MASK 0x00001fff
 
 /* vtc generator/detector vblank/vsync horizontal offset registers */
-#define VTC_XVXHOX_HEND_MASK	0x1fff0000	/* horizontal offset end */
-#define VTC_XVXHOX_HEND_SHIFT	16		/* horizontal offset end
-						   shift */
-#define VTC_XVXHOX_HSTART_MASK	0x00001fff	/* horizontal offset start */
+#define VTC_XVXHOX_HEND_MASK	0x1fff0000	/* horiz offset end */
+#define VTC_XVXHOX_HEND_SHIFT	16		/* horiz offset end shift */
+#define VTC_XVXHOX_HSTART_MASK	0x00001fff	/* horiz offset start */
 
 /* reset register bit definition */
 #define VTC_RESET_RESET		(1 << 31)	/* Software Reset */
@@ -150,11 +149,9 @@
 
 #define VTC_IXR_ACL	(1 << 21)	/* active chroma signal lock */
 #define VTC_IXR_AVL	(1 << 20)	/* active video signal lock */
-#define VTC_IXR_HSL	(1 << 19)	/* horizontal sync signal
-						   lock */
+#define VTC_IXR_HSL	(1 << 19)	/* horizontal sync signal lock */
 #define VTC_IXR_VSL	(1 << 18)	/* vertical sync signal lock */
-#define VTC_IXR_HBL	(1 << 17)	/* horizontal blank signal
-						   lock */
+#define VTC_IXR_HBL	(1 << 17)	/* horizontal blank signal lock */
 #define VTC_IXR_VBL	(1 << 16)	/* vertical blank signal lock */
 
 /* mask for all interrupts */
-- 
2.31.1

